opentelemetry-sdk/src/metrics/reader.rs

```diff
@@ -136,10 +117,9 @@ impl DefaultAggregationSelector {
     pub fn new() -> Self {
         Self::default()
     }
-}
 
-impl AggregationSelector for DefaultAggregationSelector {
-    fn aggregation(&self, kind: InstrumentKind) -> Aggregation {
+    /// Default aggregation for instrument kind
+    pub fn aggregation(&self, kind: InstrumentKind) -> Aggregation {
```

Comment:
We don't want this method to be public. In fact, since this is only used in `pipeline`, we could simply remove the struct `DefaultAggregationSelector` along with its methods and move this to `pipeline.rs` as a module level function similar to `aggregate_fn` and `is_aggregator_compatible`.